Templates/DefaultProject/Template/Gem/gem.json

```diff
@@ -0,0 +1,21 @@
+{
+    "gem_name": "${Name}",
+    "display_name": "${Name}",
+    "license": "What license ${Name} uses goes here: i.e. Apache-2.0 or MIT",
+    "license_url": "Link to the license web site goes here: i.e. https://opensource.org/licenses/Apache-2.0 Or https://opensource.org/licenses/MIT",
+    "origin": "The name of the originator goes here. i.e. XYZ Inc.",
+    "origin_url": "The primary repo for ${Name} goes here: i.e. http://www.mydomain.com",
+    "type": "",
+    "summary": "A short description of ${Name}.",
+    "canonical_tags": [
+        "Gem"
+    ],
+    "user_tags": [
+        "${Name}"
+    ],
+    "icon_path": "preview.png",
+    "requirements": "",
+    "documentation_url": "",
+    "dependencies": [
+    ]
+}
```

Comment:
Why are we going back to making projects have a Gem folder?

Reply:
Is this a breaking change for all existing projects that don't have a Gem folder? Will their folder structure need to be changed?

Reply:
No it should not be a breaking change. They can change if they wish but it is not needed.

Reply:
Were you able to configure and compile a pre-existing external project after making these changes?

Reply:
This is a template so it has nothing to do with per-existing projects. AR builds AutomatedTesting which is a preexisting project. I'll delete my AutomatedTesting, reconfigure and build. Should only take a few minutes...

Reply:
-- Configuring done
-- Generating done

Reply:
My comment about configuring and compiling pre-existing external projects doesn't pertain to this template code, but rather the .cmake changes.  The external projects I was thinking of were projects other teams/users have made that are not tested by AR like o3de-atomtest and o3de-multiplayersample.

Reply:
built.